install/fedora-installation-guide.md

```diff
@@ -2,6 +2,14 @@
 
 1. Install the Kata Containers components with the following commands:
 
+   1.1. From Fedora 31 onwards:
+
+   ```bash
+   $ sudo dnf -y install kata-runtime kata-proxy kata-shim
```

Comment:
afaik, you can just install the Fedora `kata-runtime` package which will auto-install the proxy and shim.

However, does that give you a working system? The OBS packages we produce also provide a container kernel and the disk image / initramfs files (`kata-linux-container` and `kata-containers-image` packages (see http://download.opensuse.org/repositories/home:/katacontainers:/releases:/x86_64:/master/Fedora_30/x86_64/)) but I'm not sure you get that with F31 (yet)?

/cc @jcvenegas, @lsm5.

Reply:
Aha! Okay, so, my proposal is to close this MR (and the issue associated) and get back to this one whenever Fedora has all the packages in place.

Does that work for everyone involved?

Reply:
@c3d Isn't F31 shipping the a rootfs and kernel (Fedora based) as well?

Reply:
Interesting. When taking a look at https://copr.fedorainfracloud.org/coprs/ddd/kata/ I see the kata-containers-image. However, seems that @c3d has decided to focus on the other packages for Fedora 31.

Reply:
In Fedora, kata-runtime pulls in kata-osbuilder. When kata-osbuilder is installed, rpm %post runs on the host, and calls kata-osbuilder build scripts, which invoke dracut on the host which builds an 'image' from installed host content. Some custom packaging bits also cp current host kernel/initrd from /boot into /usr/share/kata-containers and symlinks them at the expected paths for configuration.toml to pick up.

The kata-linux-container/kata-containers-image way of doing things doesn't really work with Fedora packaging policies, so we went with this approach. It's not impossible in the future we would ship a stripped down Fedora kernel specifically for kata but it would be unprecedented in Fedora land.